dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -1075,6 +1084,9 @@ protected void processMap(Contentlet contentlet, Map<String,Object> map) throws
 			if(st!=null && InodeUtils.isSet(st.getInode())) {
 				// basic data
 				contentlet.setStructureInode(st.getInode());
+				if(!UtilMethods.isSet(contentlet.getStructureInode())){
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "getStructureInode"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)